platforms/web/scripts/i18next/parse-translations.ts

```diff
@@ -0,0 +1,66 @@
+import { existsSync, readdirSync, writeFileSync } from 'node:fs';
+import { join } from 'node:path';
+import { execSync } from 'node:child_process';
+
+import prettier from 'prettier';
+
+const i18nextParserConfig = join(process.cwd(), 'i18next-parser.config.js');
+const platformGlob = join(process.cwd(), 'src/**/*.{ts,tsx}');
+
+type WorkspaceEntry = {
+  location: string;
+  workspaceDependencies: string[];
+};
+
+type WorkspaceInfo = Record<string, WorkspaceEntry>;
+
+function syncTranslations() {
+  const result = execSync('yarn workspaces --json info').toString('utf-8');
+  const info = JSON.parse(result);
+  const data = JSON.parse(info.data) as WorkspaceInfo;
+
+  const dependencies = data['@jwp/ott-web']?.workspaceDependencies || [];
```

Comment:
Do we need to inspect dependencies recursively?

Reply:
Good point. The `@jwp/ott-ui-react` package may depend on a `@jwp/ott-ui-web` package which is not listed in the platform dependencies.